indico/modules/events/contributions/templates/management/contrib_report_filter.html

```diff
@@ -0,0 +1,106 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+
+{% macro _render_column_selector(item, filter_choices) %}
+    {% set filters = filters['items'] %}
+    <div class="i-label report-column {%- if filter_choices %} filter{% endif %}"
+         data-id="{{ item.personal_data_type.name if item.personal_data_type else item.id }}">
+        <a class="title {% if filter_choices %}icon-filter{% endif %}" data-toggle="dropdown">
+            {{- item.title -}}
+        </a>
+        {% if filter_choices %}
+            <ul class="dropdown" style="display: none">
+                {% for value, caption in filter_choices.iteritems() %}
+                    <li>
+                        <input type="checkbox" name="field_{{ item.id }}" value="{{ value }}"
+                               id="field_{{ item.id }}_option_{{ value }}"
```

Comment:
shit, the registration template will have to be updated too then :stuck_out_tongue: